hail/src/main/scala/is/hail/methods/LocalLDPrune.scala

```diff
@@ -281,7 +281,7 @@ object LocalLDPrune {
   ): Table = {
     val pruner = LocalLDPrune(callField, r2Threshold, windowSize, maxQueueSize)
     val mv = mt.value
-    new Table(HailContext.get, TableLiteral(pruner.execute(mt.value)))
+    new Table(HailContext.get, TableLiteral(pruner.execute(mv)))
```

Comment:
Sorry, could we just do execute(mt.value), given that the assigned value isn’t reused?

It may not be optimized away 
https://stackoverflow.com/questions/43390869/does-python-optimize-away-a-variable-thats-only-used-as-a-return-value , and intentions seem clearer since assignment often indicates repeated use.